packages/core/src/header-builder/headers-util.ts

```diff
@@ -73,3 +73,19 @@ export function replaceDuplicateKeys(
     )
     .reduce((replaced, header) => ({ ...replaced, ...header }), {});
 }
+
+/**
+ * Create a header object by merging two header objects, where the custom headers take precedence. .
+ * @param headers - A base header object that contains the headers that will be compared with `customHeaders`.
+ * @param customHeaders - A header object to be compared with headers. Only headers present in `headers` will be compared.
+ * @returns - An object containing all keys from both the header objects, where headers present in the `customHeaders` are replaced. Note that the case (upper / lower) used by `customHeaders` will be used.
+ */
+export function mergeHeaders(
```

Comment:
Could this methed be used in the [the odata-request](https://github.com/SAP/cloud-sdk/blob/defe3b7e48abab45b225c9e829d2dc3d1c59599b/packages/core/src/odata/common/request/odata-request.ts#L197) and also other places where I see the replaceDuplicateKeys?

Reply:
Forget this comment. I was not checking the end.